src/test/resources/doc/examplesExpected/configChildren.json

```diff
@@ -0,0 +1,221 @@
+            "name":"Level1",
+            "fullName":"org.frankframework.frankdoc.testtarget.examples.config.children.Level1",
+            "elementNames":[
+                "Level1A"
+            ],
+            "attributes":[
+                {
+                    "name":"active",
+                    "description":"If defined and empty or false, then this element and all its children are ignored"
+                }
+            ],
+            "children":[
+                {
+                    "multiple":true,
+                    "roleName":"a",
+                    "type":"org.frankframework.frankdoc.testtarget.examples.config.children.IChild"
+                },
+                {
+                    "multiple":true,
+                    "roleName":"b",
+                    "type":"org.frankframework.frankdoc.testtarget.examples.config.children.IChild"
+                }
+            ]
+        },
+        {
+            "name":"Level2",
+            "fullName":"org.frankframework.frankdoc.testtarget.examples.config.children.Level2",
+            "parent":"org.frankframework.frankdoc.testtarget.examples.config.children.Level1",
+            "elementNames":[
+                "Level2A"
+            ],
+            "children":[
+                {
+                    "multiple":true,
+                    "roleName":"c",
+                    "type":"org.frankframework.frankdoc.testtarget.examples.config.children.IChild"
+                }
+            ]
+        },
+        {
+            "name":"Level3",
+            "fullName":"org.frankframework.frankdoc.testtarget.examples.config.children.Level3",
+            "parent":"org.frankframework.frankdoc.testtarget.examples.config.children.Level2",
+            "elementNames":[
+                "Level3A"
+            ],
+            "children":[
+                {
+                    "multiple":true,
+                    "roleName":"a",
+                    "description":"This comment makes it documented. Not a technical override so no reuse of declaration in {@link Level1}",
+                    "type":"org.frankframework.frankdoc.testtarget.examples.config.children.IChild"
+                },
+                {
+                    "multiple":true,
+                    "roleName":"d",
+                    "type":"org.frankframework.frankdoc.testtarget.examples.config.children.IChild"
+                }
+            ]
+        },
+        {
+            "name":"LevelWithCumulative",
+            "fullName":"org.frankframework.frankdoc.testtarget.examples.config.children.LevelWithCumulative",
+            "parent":"org.frankframework.frankdoc.testtarget.examples.config.children.Level3",
+            "elementNames":[
+                "LevelWithCumulativeA"
+            ],
+            "children":[
+                {
+                    "multiple":true,
+                    "roleName":"a",
+                    "type":"org.frankframework.frankdoc.testtarget.examples.config.children.IChild2"
```

Comment:
I did not check whether this works in the Frank!Doc webapp. We do not override a child here, but we overload it. This case is not so important though, because role names are almost always unique amongst the config children in an inheritance hierarchy.